components/exchange/TranslationResult.js

```diff
@@ -87,20 +87,9 @@ const AddToDeckContainer = styled.div`
   :hover {
     opacity: 50%;
   }
-`
-
-const AddToDeckButton = styled.div`
```

Comment:
Was there a reason this existed? It seems like it works just the same if I remove it, but maybe I am missing something? @adcaelum